softlayer/vm/softlayer_vm.go

```diff
@@ -67,6 +76,55 @@ func (vm SoftLayerVM) Delete() error {
 		return bosherr.WrapError(nil, "Did not delete SoftLayer VirtualGuest from client")
 	}
 
+	totalTime := time.Duration(0)
```

Comment:
Let's move lines 79 to 127 to it's own private helper method. Or if type of wait/check is comment then move to a utility Public method, just like I did for: `bslcommon.WaitForVirtualGuestToHaveNoRunningTransactions`